docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
+Mediation factors for GBD 2019 and 2021 can be found `on this HUB page <https://hub.ihme.washington.edu/display/GBD2020/GBD+2021+Risk+factors+mediation>`_.
+GBD mediation factors do not vary by location, age, sex, or year.
+They are not allowed to exceed 1; if the calculated value is greater than 1 it
+will be truncated to exactly 1 (this truncation happens at the draw level).
+Some mediation pathways have a mediation factor of exactly 1 by definition, because
+there is no direct effect.
+For example, lead exposure's effect on ischemic heart disease is entirely mediated by
+high systolic blood pressure.
+Draws of the mediation factor are allowed to be negative, which represents the direct pathway
+and the mediated pathway having different effect directions.
+However, if more than 50 of the 1,000 draws were negative, that mediation
+pathway was excluded from GBD.
+
+.. note::
+  Excluding when more than 50 of the 1,000 draws are negative is a kind of "significance" test.
+  However, it assumes that a significant *negative* mediation factor
+  (direct and mediated pathways have opposite effect directions) could never occur;
+  therefore GBD never models such mediation situations.
+
+This "mediation factor" can be used to calculate values for each of the causal effects represented by arrows
+in the diagram in one of two ways; the following sections describe how to do this.
+
+Unfortunately, GBD mediation factors are inconsistent with the multiplicative application of RRs.
+This is because mediation factors are defined as proportions of the *excess* risk -- the amount the RR is
+greater than 1.
+In effect, GBD's mediation methods can split e.g. an RR of 2.2 into an RR of 1.7 and an RR of 1.6 (since 0.7 + 0.6 = 1.2, which
+is the overall RR's excess risk).
+However, if we apply these two RRs, we end up with an RR of of 1.7 * 1.6 = 2.72, which is higher than
+we started with.
+For this reason, we have to decide whether to be consistent with how we apply RRs
+(and therefore stay calibrated to overall prevalence),
+or to be consistent with GBD's mediation methodology.
+In past models, we have mostly done the latter; we recommend doing the former in future models.
+Both approaches are described below.
+
+.. todo::
+
+  We don't totally understand why GBD models *excess* risk -- while there is a possibility that
+  there is a good reason behind it, we haven't been able to find that reason, so we prefer
+  Approach 1 for now.
+
+If GBD does not report a mediation factor, it may still report one or more of the effects in the diagram.
+The rest can be filled in with external sources.
+
+.. note::
+  Be careful not to interpret GBD RRs as *direct* RRs -- unless there is a mediation factor,
+  the GBD RR will not be adjusted for any mediators.
+
+All our multiplicative math is based on this equation describing how RRs are applied:
+
+.. math::
+
+  RR_\text{r→o} = RR_\text{r→o,direct} * \prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}
```

Comment:
```suggestion
  RR_\text{r→o} = RR_\text{r→o,direct} \times \prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}
```
Or use `\cdot` instead of `\times`. In math, `*` is only ever used for operations that are different from ordinary multiplication, e.g., convolution.

Also, is $RR_\text{m→o}$ actually a relative risk as we usually define it, or is it a log relative risk? I think I've run into this before with GBD terminology, and I find it confusing.

Also, it's not immediately obvious why this equation should make sense. Why do we multiply all the RRs regardless of what the causal diagram looks like? Why is the direct effect singled out as different from the others? Are these all adjusted RRs, or what?

Reply:
👍 I think a diagram could help here. This is meant to imply a single causal diagram: R -> M{1-n} -> O with a direct arrow R -> O as well.

$RR_\text{m→o}$ is a normal (not log) relative risk per unit increase in m.

We multiply the RRs of each mediation _pathway_ (the RR of R on O _through_ that mediator) with the direct RR, since that is how they get applied. The RR through the pathway is $(RR_\text{m→o})^{\Delta_\text{r→m}}$ because the total RR of the mediator on the outcome, to the power of the number of units that mediator increases for each unit increase in R.